internal/app.go

```diff
@@ -232,6 +246,42 @@ OPTIONS:
 	return app
 }
 
+func showVersion(cacheDir, outputFormat, version string, outputWriter io.Writer) {
+	db.Init(cacheDir)
+	metadata, err := db.Config{}.GetMetadata()
+	if err != nil {
+		fmt.Fprintf(outputWriter, "unable to display current version: %s", err.Error())
+		return
+	}
+	switch outputFormat {
+	case "json":
+		b, _ := json.Marshal(ttypes.VersionInfo{
+			Trivy: version,
```

Comment:
```suggestion
			Trivy: c.App.Version,
```
You have to pass c.App.Version to this function.

Reply:
I already do this https://github.com/aquasecurity/trivy/pull/435/files#diff-15cf00305be231c6aa506ff45d63ba69R187 what's missing?

Reply:
Sorry, I didn't notice.